examples/activation_maximization_colab.ipynb

```diff
@@ -1,303 +1,405 @@
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "#### 4-3. `GradientAscent.visualize`: return image tensor\n",
-    "\n",
-    "If you want to grab the optimized image data, set `return_output` to `True`."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "output = g_ascent.visualize(conv5_1, 3, title='conv5_1 filter 3', return_output=True);\n",
-    "\n",
-    "print('num_iter:', len(output))\n",
-    "print('optimized image:', output[-1].shape)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "#### 4-4. `GradientAscent.deepdream`: create DeepDream\n",
-    "\n",
-    "You can create a [DeepDream](https://ai.googleblog.com/2015/06/inceptionism-going-deeper-into-neural.html) by supplying a path to your own image.\n",
-    "\n",
-    "This will optimize the supplied image, instead of a random input noise, with regards to the filter specidied."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Download the example image\n",
-    "\n",
-    "!mkdir -p images\n",
-    "\n",
-    "!wget https://github.com/MisaOgura/flashtorch/raw/master/examples/images/jay.jpg -P /content/images"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "g_ascent.deepdream('/content/images/jay.jpg', conv5_1, 33)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "#### 4-5. `GradientAscent.optimize`: perform optimization only (no visualization)\n",
-    "\n",
-    "If no visualization is needed, or if you want to futher customize visualization, you can call the `optimize` method directly."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "output = g_ascent.optimize(conv5_1, 3)\n",
-    "\n",
-    "print('num_iter:', len(output))\n",
-    "print('optimized image:', output[-1].shape)"
-   ]
-  }
- ],
- "metadata": {
-  "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
-   "name": "python3"
-  },
-  "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.7.4"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 2
-}
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/vainaijr/flashtorch/blob/master/examples/activation_maximization_colab.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
Hi @vainaijr,

Please change the link to point to `https://colab.research.google.com/github/MisaOgura/flashtorch/blob/master/examples/activation_maximization_colab.ipynb` as discussed.

Many thanks,
Misa

Reply:
the default colab generated notebook, adds a open in colab button linking to file in my repo, so I changed both files from jupyter notebook., I added link to activation_maximization_colab in both